### PR TITLE
Feature/minimal example

### PR DESCRIPTION
**Pull request by NoB0**
Minimal example to use the demo.

- Material related to personalization has been removed
- Documentation was added for scripts and some actions

### CODE REVIEW

File: actions/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Actions
+
+Below you can information related to the different actions with indications on what should be changed to fit your use case.
```
Comment (IKostric) [inline actions/README.md]: Below you can find information ...

File: actions/maps.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            location=destination_latlong,
+            popup=destination,
+            icon=folium.Icon(color="red"),
+        )
+        # add the circle marker to the map
+        start_marker.add_to(shortest_route_map)
+        end_marker.add_to(shortest_route_map)
+
+        shortest_route_map.save("ui/furhat-screen/assets/map/route.html")
+        return "assets/map/route.html"
```
Comment (IKostric) [inline actions/maps.py]: Maybe have these two strings as constants on top of the file.

File: ui/furhat-screen/assets/map/route.html [hunk 3]
```diff
@@ -0,0 +1,542 @@
+<!DOCTYPE html>
```
Comment (IKostric) [inline ui/furhat-screen/assets/map/route.html]: Is this automatically generated file? If so, we do not need it in the repository.

File: ui/furhat-skills/rasa/src/main/kotlin/furhatos/app/rasa/flow/interaction.kt [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+import furhatos.event.Event
+
+import java.util.*
+import furhatos.event.EventSystem
+
+import java.util.regex.Pattern
+
+// CHANGE IP AND PORT HERE:
+val IP = "192.168.151.111"
+// val IP = "10.11.0.157"
```
Comment (IKostric) [inline ui/furhat-skills/rasa/src/main/kotlin/furhatos/app/rasa/flow/interaction.kt]: Replace the IP with "localhost" and remove the commented line.

File: ui/webchat/index.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+     
+</head>
+ 
+<body>   
+    <div id="chatwidget">
+    <script>
+    // CHANGE IP AND PORT HERE:
+    const IP = "10.192.201.21";
```
Comment (IKostric) [inline ui/webchat/index.html]: Replace the IP with "localhost".

File: README.md [hunk 6]
```diff
@@ -81,7 +81,18 @@ rasa shell nlu
 
 ## Webchat
 
-TBD
+You can start a chat webserver as follows. If you do not use the default port to run the Rase server, you need to change the port in this [file](ui/furhat-screen/index.js).
```
Comment (IKostric) [inline README.md]: Rase --> Rasa

File: docs/how-to-run.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+You change IP at the top of the file in question.
+You change the IP in 
+> dagfinn\ui\furhatScreen.index.js
+
+and in
+
+>dagfinn\ui\furhat-skills\rasa\src\main\kotlin\furhatos\app\rasa\flow.interaction.kt
+
+```
+const IP = "10.192.201.55"     // or  "localhost"
```
Comment (IKostric) [inline docs/how-to-run.md]: Only localhost.

**Review by IKostric — APPROVED** [review]
> Minimal example to use the demo.

- Material related to personalization has been removed
- Documentation was added for scripts and some actions  

Can you remove the conversational flow in `data/pers`? 

There are some minor comments, othrwise LGTM.

**Comment by IKostric** [comment]
> Minimal example to use the demo.

- Material related to personalization has been removed
- Documentation was added for scripts and some actions  

Can you remove the conversational flow in `data/pers`? 

There are some minor comments, othrwise LGTM.

File: ui/furhat-screen/assets/map/route.html (hunk 3 above)
Comment (NoB0) [reply ui/furhat-screen/assets/map/route.html]: File removed.

File: ui/furhat-skills/rasa/src/main/kotlin/furhatos/app/rasa/flow/interaction.kt (hunk 4 above)
Comment (NoB0) [reply ui/furhat-skills/rasa/src/main/kotlin/furhatos/app/rasa/flow/interaction.kt]: Done.

File: ui/webchat/index.html (hunk 5 above)
Comment (NoB0) [reply ui/webchat/index.html]: Done.

File: README.md (hunk 6 above)
Comment (NoB0) [reply README.md]: Done.

File: docs/how-to-run.md (hunk 7 above)
Comment (NoB0) [reply docs/how-to-run.md]: Done.

**Pull request merged**